### PR TITLE
[Misc] Reorder the variables in minibatch.py.

### PR DESCRIPTION
**Pull request by frozenbugs**
## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] I've leverage the [tools](https://docs.google.com/document/d/1iHyj7zlmygKSk5gBPsqIqL5ASPzJSPREaNT_QdsiYA4/edit) to beautify the python and c++ code.
- [ ] The PR is complete and small, read the [Google eng practice (CL equals to PR)](https://google.github.io/eng-practices/review/developer/small-cls.html) to understand more about small PR. In DGL, we consider PRs with less than 200 lines of core code change are small (example, test and documentation could be exempted).
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the best of my knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

File: python/dgl/graphbolt/minibatch.py [hunk 1]
```diff
@@ -85,6 +42,15 @@ class MiniBatch:
       type.
     """
 
+    labels: Union[torch.Tensor, Dict[str, torch.Tensor]] = None
+    """
+    labelss associated with seed nodes in the graph.
```
Comment (Rhett-Ying) [inline python/dgl/graphbolt/minibatch.py:47]: please help fix the typo `labelss` I introduced

**Review by Rhett-Ying — APPROVED** [review]

File: python/dgl/graphbolt/minibatch.py (hunk 1 above)
Comment (frozenbugs) [inline python/dgl/graphbolt/minibatch.py:47]:
```suggestion
    labels associated with seed nodes in the graph.
```

File: python/dgl/graphbolt/minibatch.py [hunk 2]
```diff
@@ -85,6 +42,15 @@ class MiniBatch:
       type.
     """
 
+    labels: Union[torch.Tensor, Dict[str, torch.Tensor]] = None
+    """
+    labels associated with seed nodes in the graph.
```
Comment (frozenbugs) [inline python/dgl/graphbolt/minibatch.py:47]:
```suggestion
    Labels associated with seed nodes / node pairs in the graph.
```

File: python/dgl/graphbolt/minibatch.py [hunk 3]
```diff
@@ -85,6 +42,15 @@ class MiniBatch:
...
+    - If `labels` is a tensor: It indicates the graph is homogeneous. The value
+      should be corresponding labelss to given 'seed_nodes' or 'node_pairs'.
+    - If `labels` is a dictionary: The keys should be node or edge type and the
+      value should be corresponding labelss to given 'seed_nodes' or 'node_pairs'.
```
Comment (frozenbugs) [inline python/dgl/graphbolt/minibatch.py:51]:
```suggestion
    - If `labels` is a tensor: It indicates the graph is homogeneous. The value
      should be corresponding labels to given 'seed_nodes' or 'node_pairs'.
    - If `labels` is a dictionary: The keys should be node or edge type and the
      value should be corresponding labels to given 'seed_nodes' or 'node_pairs'.
```

**Pull request merged**